region/norfair/east/Bubble Mountain.json

```diff
@@ -1110,6 +1110,194 @@
         "canTrickyGrappleJump"
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Side Platform Cross Room Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 13,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"or": [
+                  "HiJump",
+                  {"and": [
+                    "canTrickyDashJump",
```

Comment:
Dboost doesn't need a trickydashjump but maybe that is covered somewhere already.

Reply:
Makes sense, restructured to add a dboost option now.